library/src/main/java/com/getstream/sdk/chat/utils/StringUtility.java

```diff
@@ -66,4 +68,14 @@ public static String convertVideoLength(long videoLength) {
         long seconds = videoLength % 60;
         return String.format("%02d:%02d:%02d", hours, minutes, seconds);
     }
+
+    @SuppressLint("DefaultLocale")
+    public static String convertFileSizeByteCount(long bytes) {
+        int unit = 1000;
+        if (bytes < unit) return bytes + " B";
```

Comment:
I would add also zero and negative cases. Never trust input parameters.
That's how negative unread counters appear in production.